sql/src/main/java/io/crate/executor/transport/SymbolBasedTransportShardUpsertAction.java

```diff
@@ -150,19 +150,14 @@ protected boolean checkWriteConsistency() {
     }
 
     @Override
-    protected boolean ignoreReplicas() {
```

Comment:
please check with @seut if this means that the Upsert action is now done on both primary and replica and if this causes duplicate index requests